src/services/issues.service.ts

```diff
@@ -39,14 +40,25 @@ export class IssuesService {
   async drawIssuesTable(issues: IIssueResponse[]) {
     const table = new Table();
 
-    table.push([{ colSpan: 5, content: 'PR Issues', hAlign: 'center' }]);
+    table.push([{ colSpan: 6, content: 'PR Issues', hAlign: 'center' }]);
 
-    table.push(['№', 'Name', 'Severity', 'Dependency name', 'Dependency version']);
+    table.push(['№', 'Severity', 'Name', 'Dependency', 'Category', 'Fixed Version']);
 
     issues.forEach((issue, index) => {
-      table.push([index + 1, issue.name, issue.severity, issue.dependencyName, issue.dependencyInstalledVersion]);
+      table.push(this.transformIssueToRow(issue, index));
     });
 
     console.log(table.toString());
   }
+
+  transformIssueToRow(issue: IIssueResponse, index: number) {
+    const severity = capitalize(issue.severity);
```

Comment:
why do we need to use lodash for capitalize? not a complex task

Reply:
we already have the lodash package, I prefer to use it instead of creating the same functions

Reply:
<img width="1297" alt="image" src="https://github.com/blindspot-security/myrror-cli/assets/57172661/f176e12f-695c-4df9-a05d-9d8fd437d85c">

Reply:
Lodash is already used in libs that we use, if we don't use it directly it is in our bundle anyway